META6.json

```diff
@@ -59,8 +58,7 @@
     "postgres",
     "sqlite"
   ],
-  "test-depends": [
-    "NativeCall::TypeDiag"
-  ],
-  "version": "0.6.5"
-}
+  "api": 1,
+  "source-url": "https://github.com/raku-community-modules/DBIish.git",
+  "raku": "6.c"
+}
```

Comment:
Do all 6.c versions support the new extensions? If know the raku spec version doesn't correlate with the raku compiler version but can we somehow ensure a new enough raku compiler version is required?

Reply:
6.c goes back a long way.

While DBIish don't need 6.d syntax, it is the least crashy on on interpreters which support it, so I made that change.